### ISSUE TITLE
Cosmetic - control chars related to volumes when VCH removed

### ISSUE THREAD

**Issue by cormachogan** [vmware/vic#1651]
**VIC Version:**

0.1 / master / 3190

Cosmetic issue - some control chars are displayed when VCH is removed, and a volume has been defined, but force option has not been used:

```
./vic-machine-linux delete -t 10.27.51.103 -u 'administrator@vsphere.local' -p 'VMware123!' --compute-resource Mgmt
INFO[2016-07-25T13:04:32Z] ### Removing VCH ####
INFO[2016-07-25T13:04:32Z] Removing VMs
INFO[2016-07-25T13:04:32Z] Removing images
WARN[2016-07-25T13:04:32Z] Since --force was not specified, the following volume stores will not be removed. Use the vSphere UI to delete content you do not wish to keep.
"\tdefault: [isilion-nfs-01] docker-vols\n"
INFO[2016-07-25T13:04:32Z] Removing appliance VM network devices
INFO[2016-07-25T13:04:33Z] Removing VCH vSphere extension
INFO[2016-07-25T13:04:38Z] Removing Resource Pool "virtual-container-host"
INFO[2016-07-25T13:04:38Z] Completed successfully
root@photon-NaTv5i8IA [ /workspace/vic ]#
```

**Expected behavior:**
Not control chars - remove them from output

**Actual behavior:**
As above

**Comment by gigawhitlocks** [comment]
@emlin it looks like this was fixed in PR #1536 ?

**Comment by emlin** [comment]
@gigawhitlocks Maybe post the new output here, and then it's easy to see if that's good now.

Looks like you used \n in the output, not quite sure if that's a good format. Why you need \n in the log message?

**Comment by gigawhitlocks** [comment]
@emlin the newline is there to separate hits. If you don't specify --force, it will print out each volume store that is *not* removed, for the user's reference, so the \n is to separate them. I fixed it incidentally in PR #1678

**Comment by emlin** [comment]
@gigawhitlocks then that's great 👍
